### DISCUSSION TITLE
Saving screenshots directly

### DISCUSSION THREAD

**Discussion by ppskj178** [karakeep-app/karakeep#589] [Ideas]
Ultimately, I'd love to have the ability to hit the share after screenshot button on a mobile or desktop device and save it directly to HOARDER, but if that's not possible right now, I'll suggest a few shortcuts.
(Based on galaxy and window, which I use (win + shift + s), there is a button to share after screenshot).

1. a feature that automatically uploads files (txt, img, pdf ...) in a specific path to your server
2. the ability to embed a capture tool in the web extension and save the captured image directly.

I think both of these would be relatively easy to implement.
this feature would greatly improve usability if implemented.

**Comment by MohamedBassem** [comment]
@ppskj178 If you have the mobile app, you should be able to share to hoarder directly from the "share sheet". That's the main benefit of using the app.

[https://docs.hoarder.app/img/screenshots/share-sheet.png](url)

**Reply by ppskj178** [reply]
url is not working

i want same thing in windows

**Reply by MohamedBassem** [reply]
Sorry, you can find this screenshot here: https://docs.hoarder.app/screenshots#ios-sharing
